.vscode/PythonImportHelper-v2-Completion.json

```diff
@@ -0,0 +1,2360 @@
+[
```

Comment:
what is this file for exactly?

Reply:
idk, automatically added by vs code, lemme remove it

Reply:
done